packages/backend/src/util/api-message-list.ts

```diff
@@ -0,0 +1,10 @@
+export const API_MESSAGE_LIST = {
+  GENERAL: {
```

Comment:
Maybe better to not seperate for now and create later some categories of messages